integration-tests/testsuite/src/it/java/org/graalvm/tests/integration/utils/BuildAndRunCmds.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) 2020, Red Hat Inc. All rights reserved.
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package org.graalvm.tests.integration.utils;
+
+/**
+ * BuildAndRunCmds
+ *
+ * The last command is used to run the final binary.
+ * All previous commands are used to build it.
+ *
+ * @author Michal Karm Babacek <karm@redhat.com>
+ */
+public enum BuildAndRunCmds {
+    // Note that at least 2 command are expected. One or more to build. The last one to run the app.
+    QUARKUS_FULL_MICROPROFILE(new String[][]{
+            new String[]{"mvn", "clean", "compile", "package", "-Pnative"},
```

Comment:
How does `mvn` pick `native-image` in this case?

Reply:
@zakkak It differs runtime to runtime I am afraid. Some just expect it on PATH. Quarkus looks for GRAALVM_HOME too.

The TS as it is right now is oblivious to native-image location; it just expects it on your PATH.  Maybe I should enhance the README...

Reply:
If it's going to be part of mandrel/graal rerpo one would expect it to test the latest/current build I guess, so that could be confusing.

I agree that it needs to be documented in the README, and possibly extended at a later stage to be configurable without having to change `PATH`.